package.json

```diff
@@ -19,9 +19,11 @@
     "build-teleport-e": "yarn workspace @gravitational/teleport.e build --output-path=../../../dist/e/teleport/app",
     "build-oss": "yarn build-teleport-oss",
     "build-e": "yarn build-teleport-e",
-    "type-check-and-lint": "yarn type-check && yarn lint",
+    "type-check-and-lint": "yarn type-check && yarn prettier-check && yarn lint",
```

Comment:
i figured prettier fell under the `lint` category so I put it here and i also saw that our CI runs the [type-check-and-lint](https://github.com/gravitational/webapps/blob/master/.cloudbuild/ci/build.yaml#L21), i'm not sure if this means it will run prettier now? (i need to merge the PRs that this PR bases from to see if it can be applied to this PR, like add a file to deliberately fail the check to test it)